samples/subsys/mgmt/mcumgr/smp_svr/prj_udp.conf

```diff
@@ -0,0 +1,40 @@
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Some command handlers require a large stack.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Ensure an MCUboot-compatible binary is generated.
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable the UDP mcumgr transports.
+CONFIG_MCUMGR_SMP_UDP=y
+CONFIG_NET_BUF_USER_DATA_SIZE=8
+
+# Network settings
+CONFIG_NETWORKING=y
+CONFIG_NET_MGMT=y
+CONFIG_NET_L2_ETHERNET_MGMT=y
```

Comment:
Is this UDP transport only working with Ethernet?
If yes, then the max Ethernet MTU is 1500 bytes so you would not be able to receive 2027 byte UDP packets.

Reply:
I have changed the MTU to 1500 bytes. The reason for the ~2000 bytes is that the MTU in the mcumgr cli is hardcoded to 2000 bytes for UDP transport. I have not seen any packets that are more than ~600 bytes when uploading images, so 1500 should be okay.